content/developer/reference/frontend/services.rst

```diff
@@ -341,6 +343,40 @@ Here, it is called in webclient.js to make it visible everywhere for the example
     :width: 600
     :align: center
 
+.. _frontend/services/http:
+
+Http Service
+------------
+
+Overview
+~~~~~~~~
+
+* Technical name: `http`
+* Dependencies: None
+
+While most interaction between the client and the server in odoo are `RPCs`, lower level
+control on requests may sometimes be required.
+
+This service provides a way to send `get` and `post` http requests.
+
+API
+~~~
+
+.. js:function:: async get(route [,readMethod = "json"])
```

Comment:
You'll probably want to fine-tune a bit the way the square brackets look with the new style of js docstrings.